src/formpack/utils/geojson.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+
+from __future__ import (unicode_literals, print_function, absolute_import,
+                        division)
+
+from geojson_rewind import rewind
+
+
+class FormPackGeoJsonError(Exception):
```

Comment:
OK, I see your point. For this one, since it's so trivial and only gets raised by the one class, I'd argue it makes more sense to leave it in the same file. I'll go ahead and move it, though.